fuel-core/src/model/contract.rs

```diff
@@ -0,0 +1,10 @@
+use super::Hash;
+use fuel_tx::ContractId;
+
+#[derive(Debug, Copy, Clone)]
+pub struct Contract {
```

Comment:
This is unused, and I think these fields like utxo_id will be fetched on the fly from the grapqhl resolver in other ways.

Reply:
Seems like I badly resolved a merged conflict. Will take care of it.